packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -656,7 +656,7 @@ class HotRunner extends ResidentRunner {
     for (FlutterDevice device in flutterDevices) {
       for (FlutterView view in device.views) {
         if (view.uiIsolate == null) {
-          throw 'Application isolate not found';
+          throw Exception('Application isolate not found');
```

Comment:
Should this just return an OperationResult()? Said another way, what is the reason for mixing exceptions and error returns here?

Reply:
Done